h/db/__init__.py

```diff
@@ -44,31 +45,24 @@
 Session = sessionmaker()
 
 
-def init(engine, base=Base, should_create=False, should_drop=False, authority=None):
-    """Initialise the database tables managed by `h.db`."""
-    # Import models package to populate the metadata
-    import h.models  # pylint: disable=unused-import
-
-    if should_drop:  # pragma: no cover
-        # SQLAlchemy doesn't know about the report schema, and will end up
-        # trying to drop tables without cascade that have dependent tables
-        # in the report schema and failing. Clear it out first.
-        engine.execute("DROP SCHEMA IF EXISTS report CASCADE")
```

Comment:
There's no DB initialization code in the app itself anymore. Our apps all have DB initialization code in `<APP>/db.py` or `<APP>/db/__init__.py` and this initialization code is different for each app. That's now all moved into `init_db.py` where it'll be the same for each app.